Session03/Session03/Session03Excercise02/Program.cs

```diff
@@ -1,20 +1,52 @@
 ﻿using System;
+using System.Globalization;
 
 namespace Session03Excercise02
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Ange ett antal siffror, separerat med kommatecken.");
-            var input = Console.ReadLine();
-            var inputArray = input.Split(new [] { ',' }, StringSplitOptions.RemoveEmptyEntries);
-
-            foreach (var number in inputArray)
+            IFormatProvider format = CultureInfo.CurrentCulture;
+            Console.WriteLine("Nuvarande språkinställning: " + CultureInfo.CurrentCulture.NativeName + "/" + CultureInfo.CurrentCulture.EnglishName);
+            while (true)
             {
-                Console.WriteLine("Värdet är " + number);
-            }
+                Console.WriteLine("Ange ett antal siffror, separerat med mellanslag.");
+                var input = Console.ReadLine();
+                if (input != "")
+                {
+                    var inputArray = input.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
+                    var inputTotal = 0.0;
+                    var inputLength = inputArray.Length;
 
+                    foreach (var number in inputArray)
+                    {
+                        Console.Write("Värdet är " + number);
+                        if (double.TryParse(number, out double numberParse))
+                        {
+                            inputTotal += Convert.ToDouble(numberParse);
+                        }
+                        else
+                        {
+                            //Error exception, wrong format
+                            //Set color and then reset it after the error
+                            Console.ForegroundColor = ConsoleColor.Red;
+                            Console.Write(" ERROR: FORMAT EXCEPTION");
+                            Console.ResetColor();
+                            //Fix average (without removing the entry)
+                            inputLength--;
+                        }
+                        Console.WriteLine();
+                    }
+
+                    Console.WriteLine("Total: " + inputTotal + " Average: " + Math.Round(inputTotal / (double)inputLength, 3));
```

Comment:
Saknar max och minvärde.

Reply:
Hmm, missade att man skulle ha det också. Ska jag uppdatera med det eller ska jag låta det vara för den här gången?

Jag hade troligen gjort nånting i stil med 
`if(maxValue < numberParse) { maxValue = numberParse; }`
i TryParse if-satsen på rad 25 (och med ett minValue).

Ser att jag hade nån onödig convert to double där också.

Reply:
Ja, du fattar principen, behövs ej. Jag hade skrivit numberParse > maxValue (samma sak, men min hjärna läser det lättare).